net-misc/i2pd/i2pd-2.10.1.ebuild

```diff
@@ -97,3 +88,12 @@ pkg_setup() {
 	enewgroup "${I2PD_GROUP}"
 	enewuser "${I2PD_USER}" -1 -1 /var/lib/run/i2pd "${I2PD_GROUP}"
 }
+
+pkg_postinst() {
+	if [[ -f ${ROOT}etc/i2pd/subscriptions.txt ]]; then
```

Comment:
Yeah, I have read the manual, and wondered what's the rationale for the longer version :-)